ocean_provider/routes/compute.py

```diff
@@ -116,11 +114,7 @@ def initializeCompute():
     valid_until = int(valid_until)
 
     if not timestamp_ok:
-        return error_response(
-            "The validUntil value is not correct.",
-            400,
-            logger,
-        )
+        return error_response("The validUntil value is not correct.", 400, logger)
```

Comment:
can you please make sure you use the same black version as we do? We shouldn't have any formatting-only changes.

Reply:
Done.  I reverted all formatting-only changes.  

I will mention that many of the formatting changes made sense to me.  I don't believe black has been run on all files in this repo.

Additionally, the black version specified in `.pre-commit-config.yaml` does not match the black version specified in `setup.py`